doc/man1/s_server.pod

```diff
@@ -709,6 +710,13 @@ greater than or equal to 0.
 
 Accept early data where possible.
 
+=item B<-no_anti_replay>
+
+Switches off replay protection. With replay protection on OpenSSL will
+automatically detect if a session ticket has been used more than once, TLSv1.3
+has been neotiated and early data is enabled on the server. A full handshake is
```

Comment:
"negotiated".  (And always feel free to toss in a serial comma!)

Reply:
Fixed